src/components/TreeMapControls/index.tsx

```diff
@@ -89,6 +90,25 @@ export const TreeMapControls: FC<TreeMapControlsPropType> = ({
             })}
             additionalClasses="z-0"
           />
+          <div className="hidden sm:inline-flex">
+            <Separator />
+          </div>
+          <ListBox
```

Comment:
I'd rather use a SwitchToggle here, as there are only two options.

Reply:
Thank you for die review!
1. I deleted the Husky files by mistake. I have now added them in again.
2. True. I created an issue for the suggestion with the filter. For now, I've made the bars in the mobile view a little narrower so they don't take up quite so much space.
3. There is theoretically a third possibility to order the data, by the "group". If you want to implement this sometime in the future, you would need the list again. That's why I'll leave it at that for now.
4. If you hide the district select (I think there is a typo in the review, you mean the "district" not "Einzelpläne" right?) then the districts can no longer generate an "embed" link only for their district. i have not changed anything here because of this. but i am also not quite sure if i have understood this correctly.

Reply:
1. Thx!
2. Thx too! That's better
3. I'm fine with that then
4. Regarding the district select, what I meant is that 2 filter combinations produce the same thing:
  - When "Mitte" is selected and "Funktionen". 
  - When "Alle Bereiche" is selected and "Einzelpläne" + "Mitte" is clicked. 
  Do you know what I mean? In other words, if you selected "Einzelpläne", it makes no sense to filter additionally by district because you can click on a district in the visualisation itself and thus produce the same output.